apps/condo/domains/contact/components/ContactsEditor/index.tsx

```diff
@@ -211,7 +211,7 @@ export const ContactsEditor: React.FC<IContactEditorProps> = (props) => {
                                             displayMinusButton={true}
                                             onClickMinusButton={handleClickOnMinusButton}
                                         />
-                                        {!role.canManageContacts && (
+                                        {(!role || !role.canManageContacts) && (
```

Comment:
It can be simplified with `get` from `lodash`:

```suggestion
                                        {(!get(role, 'canManageContacts')) && (
```

Reply:
Ah, — "client error". Sorry, it's correct.